src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.data;
+
+public final class User {
+
+  private final String name;
+  private final String email;
+  private final String wishlist;
+  private final String teachlist;
+
+  public User(String name, String email, String wishlist, String teachlist) {
+    this.name = name;
+    this.email = email;
+    this.wishlist = wishlist;
+    this.teachlist = teachlist;
+  }
+
+  public String toString(){
+    return "name: "+name+", email: "+email+ ", wishlist: " + wishlist + ", teachlist: " + teachlist;
+  }
```

Comment:
What is this for? It looks kinda like JSON, in which case you could straight up use Gson to get a string.

Also this is not properly formatted, did you set up prettier properly?